pkg/auth/google.go

```diff
@@ -18,18 +18,18 @@ type Google struct {
 }
 
 const (
-	googleDiscoveryURL = "https://www.googleapis.com/oauth2/v3/certs"
-	googleIssuer       = "https://accounts.google.com"
+	jwksEndpoint = "https://www.googleapis.com/oauth2/v3/certs"
```

Comment:
```suggestion
	googleJwksEndpoint = "https://www.googleapis.com/oauth2/v3/certs"
```

Siden vi har flere providers i samme pakke, så tenker jeg vi bør namespace konstanten. Trenger endringer under også dersom du er enig :)

Reply:
ja, de tenkte jeg ikke på :) bra sett